src/components/MentorApplication/MentorApplication.component.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { type Mentor } from '../../types';
+
+interface MentorApplicationProps {
+  mentor: Mentor | undefined;
+  onApprove: (mentorUuid: string) => void;
+  onReject: (mentorUuid: string) => void;
+}
+
+const MentorApplication: React.FC<MentorApplicationProps> = ({
+  mentor,
+  onApprove,
+  onReject,
+}) => {
+  const handleApprove = () => {
+    if (mentor != null) {
+      onApprove(mentor.uuid);
+    }
+  };
+
+  const handleReject = () => {
+    if (mentor != null) {
+      onReject(mentor.uuid);
+    }
+  };
```

Comment:
Can you integrate the API here? it's `admin/mentors/:mentorId/status` with the status on the body